src/splitInteger.test.js

```diff
@@ -1,21 +1,55 @@
 'use strict';
 
 const splitInteger = require('./splitInteger');
+// const { parts } = require('./splitInteger');
 
-test(`should split a number into equal parts
-  if a value is divisible by a numberOfParts`, () => {
-
+test(`should split a number into equal parts if a value
+ is divisible by  a numberOfParts`, () => {
+  expect(splitInteger(16, 4)).toEqual([4, 4, 4, 4]);
 });
 
 test(`should return a part equals to a value
   when splitting into 1 part`, () => {
+  const mas = [4];
 
+  expect(splitInteger(4, 1).length).toEqual(mas.length);
 });
 
-test('should sort parts ascending if they are not equal', () => {
-
+test('should return corect equal', () => {
+  expect(splitInteger(8, 1)).toEqual([8]);
+  expect(splitInteger(6, 2)).toEqual([3, 3]);
+  expect(splitInteger(17, 4)).toEqual([4, 4, 4, 5]);
+  expect(splitInteger(32, 6)).toEqual([5, 5, 5, 5, 6, 6]);
```

Comment:
The first 3 asserts are parts of other tests. No need to duplicate tests. In general, we check the correct values in each test. So, this test is excessive